src/test/java/io/jenkins/plugins/forensics/miner/CodeMetricTrendChartTest.java

```diff
@@ -0,0 +1,75 @@
+package io.jenkins.plugins.forensics.miner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.echarts.Build;
+import edu.hm.hafner.echarts.BuildResult;
+import edu.hm.hafner.echarts.ChartModelConfiguration;
+import edu.hm.hafner.echarts.LinesChartModel;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests CodeMetricTrendChart.
```

Comment:
Not all IDEs support refactoring in comments:

```suggestion
 * Tests the class {@link CodeMetricTrendChart}.
```